mediapipe/tasks/ios/test/vision/hand_landmarker/utils/sources/MPPHandLandmarkerResult+ProtobufHelpers.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "mediapipe/tasks/ios/vision/hand_landmarker/sources/MPPHandLandmarkerResult.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@interface MPPHandLandmarkerResult (ProtobufHelpers)
+
++ (MPPHandLandmarkerResult *)
+    handLandmarkerResultFromTextEncodedProtobufFileWithName:(NSString *)fileName
```

Comment:
Note that this is not necessarily a text proto since we use binary protos internally.

Will fix during import.